operations/mimir-rules-action/entrypoint.sh

```diff
@@ -106,8 +106,7 @@ case "${ACTION}" in
     ;;
 esac
 
-SINGLE_LINE_OUTPUT=$(echo "${OUTPUT}" | awk 'BEGIN { RS="%0A" } { gsub(/%/, "%25"); gsub(/\r/, "%0D"); gsub(/\n/, "%0A") } { print }')
-echo "detailed=${SINGLE_LINE_OUTPUT}" >> "${GITHUB_OUTPUT}"
+{ echo "detailed<<EOF"; echo "$OUTPUT"; echo "EOF"; } >> "$GITHUB_OUTPUT"
```

Comment:
Is there an equal missing?
```suggestion
{ echo "detailed=<<EOF"; echo "$OUTPUT"; echo "EOF"; } >> "$GITHUB_OUTPUT"
```

Reply:
I'm confused because the summary below is also missing the equal 🤔

Reply:
Okay, I've read the docs and this seems to be correct 👍